pinc/genre_table.inc

```diff
@@ -0,0 +1,28 @@
+<?php
+include_once($relPath.'genres.inc');
+
+function create_genre_table()
```

Comment:
Rename this `create_temporary_genre_translation_table()` and move this into `pinc/genres.inc` and call it right before we need it (in `_search()` and `_load_project_filter_field_values()`, etc). The function can check if the table exists and if not, it creates it with the logic you have here. This way we try to keep the table creation closer to the code that is going to use it.

Reply:
I've also increased genre size to VARCHAR(50) because that is what it is in the projects table.
We could make a Genre class instead of using the global variable $GENRES. But $GENRES is used in a noncvs file so I won't do it now. (How would we synchronize such a change? presumably move the file out of noncvs first?)

Reply:
I couldn't check if the table exists elegantly:
It looks like CREATE TABLE IF NOT EXISTS still creates a table but doesn't issue a warning,
SHOW TABLES and the INFORMATION_SCHEMA TABLES Table does not list temporary tables,
INFORMATION_SCHEMA INNODB_TEMP_TABLE_INFO Table needs PROCESS privilege.
In the end just tried to access the table and caught exception if doesn't exist.